docs/LwM2M_Client/OpenThread/OTBR_with_svetovid.md

```diff
@@ -0,0 +1,183 @@
+# OpenThread Border Router configurable through LwM2M server
+
+In this tutorial, we will configure OpenThread Border Router with [**Svetovid**](https://github.com/AVSystem/Svetovid-raspberry-client) which is an LwM2M Client for Linux-based devices based on [**Anjay**](https://anjay.io).
+With the help of **Svetovid**, we will be able to manage the Border Router remotely, through the LwM2M server. Within this guide, we will use the {{ coiote_long_name }} as the server.
+
+## Prerequisites
+
+- The [**nRF52840 DK**](https://www.nordicsemi.com/Products/Development-hardware/nrf52840-dk) or the [**nRF52840 Dongle**](https://www.nordicsemi.com/Products/Development-hardware/nrf52840-dongle).
+- A Linux-based machine (in this tutorial we will use Raspberry Pi 3 model B with Raspbian) with `AMD64`, `ARMv7` or `ARM64` architecture.
+- Installed [**Docker Engine**](https://docs.docker.com/engine/install/).
+- Installed **nrfjprog** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download) (for **nRF52840 DK**)
+- Installed **nrfutil** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nRF-Util) (for **nRF52840 Dongle**)
+- A user with access to the {{ coiote_long_name }}.
+
+## Flash Radio Co-Processor
+
+Our Border Router will run on an RCP design, which means that the core of the OpenThread stack will run on the host side and communicate via `Spinel` protocol with another device with a Thread radio.
+
+Now we will build and flash image for Radio Co-Processor.
+
+0. Connect your nRF board to your computer. In case of **nRF52840 DK** you should choose `MCU USB` port.
+
+0. Clone the OpenThread repository:
+```
+    git clone --recursive https://github.com/openthread/ot-nrf528xx.git
+```
+
+0. Enter new directory:
+```
+    cd ot-nrf528xx
+```
+
+0. Install required dependencies:
```

Comment:
Or sudo in general:
```
root@df11a7ed713c:/ot-nrf528xx#     script/bootstrap
++ dirname script/bootstrap
+ script/../openthread/script/bootstrap
+ main
+ install_packages
+ PM=source
+ command -v apt-get
/usr/bin/apt-get
+ PM=apt
+ install_packages_apt
+ echo 'Installing toolchain dependencies...'
Installing toolchain dependencies...
+ sudo apt-get update
script/../openthread/script/bootstrap: line 58: sudo: command not found
```

Reply:
Hmm it is probably not the only command that requires sudo privileges, and I think it would be a bit odd if I mentioned it with each of these commands.

Reply:
Maybe then state this in the Prerequisites?